vico/core/src/main/java/com/patrykandpatryk/vico/core/extension/CollectionExtensions.kt

```diff
@@ -32,6 +32,26 @@ public fun <T> MutableList<T>.setAll(other: Collection<T>) {
     addAll(other)
 }
 
+/**
+ * Replaces all of the elements in child [ArrayList]s of this [ArrayList] with the elements of the provided [List].
```

Comment:
I’d suggest making the documentation for `setToAllChildren` a bit more precise.

```suggestion
 * For each child [ArrayList] contained in this [ArrayList], replaces the elements of the child [ArrayList] with the
 * elements of the corresponding [Collection] from the provided [List]. The child [ArrayList] and [Collection] are
 * associated by index. If a given child [ArrayList] contained in this [ArrayList] has no corresponding [Collection] in
 * the provided [List], the child [ArrayList] will be cleared. If the size of this [ArrayList] is smaller than the size
 * of the provided [List], an appropriate number of empty child [ArrayList]s will first be added to this [ArrayList].
```